.github/workflows/rubocop.yml

```diff
@@ -2,16 +2,28 @@ name: Rubocop
 
 on:
   push:
-    branches: [ master ]
+    branches: [master]
   pull_request:
-    branches: [ master ]
+    branches: [master]
 
 jobs:
   rubocop:
     runs-on: ubuntu-latest
+
     steps:
-    - uses: actions/checkout@v2
-    - name: Rubocop Linter Action
-      uses: andrewmcodes/rubocop-linter-action@v3.0.0
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - uses: actions/checkout@v2
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@ec106b438a1ff6ff109590de34ddc62c540232e0
+        with:
+          ruby-version: 2.6.6
+
+      - name: Install gems
+        run: |
+          gem install rubocop
+          gem install rubocop-minitest
+          gem install rubocop-performance
+          gem install rubocop-rails
+
+      - name: Run Rubocop
+        run: rubocop --except Metrics
```

Comment:
Is this different than `rubocop --rails`?

Reply:
Yes.

In the [rubocop.yml](https://github.com/exercism/v3-website/blob/master/.rubocop.yml#L2) we just the various things it should run (rails, performance, etc).

The `--except Metrics` says "don't fail if methods are too complex etc", only if there is something that is a lint violation.

Reply:
I think it's worthwhile to make the unilateral decision to then move that to rubocop config file. It's weird if you get the errors when developing, but then not during CI.

With my comment I meant: I think `--rails` turns on some rails specific cops, that are different than `rubocop-rails`, which you might or might not want, as well.